style.css

```diff
@@ -492,9 +533,172 @@ nav {
   font-weight: 400;
   font-style: normal;
   line-height: 24px;
-  width: 327px;
+  width: 100%;
   height: 114px;
   padding: 10px;
   border: 0 solid;
   background-color: #fbf8f7;
 }
+
+#separator3 {
+  width: 100%;
+  border: 1px solid #dfe1e6;
+}
+
+#footer-socials {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  margin: 25px 0;
+}
+
+.social-icons2 {
+  font-size: 24px;
+  padding: 10px;
+  color: #505f79;
+}
+
+@media only screen and (min-width: 992px) {
```

Comment:
- [ ] I think a screen with 992 pixels is too wide for a Mobile screen size view. As previously explained in an early comment. it would be useful to use a recommended Mobile screen size of 768px instead of 992px